tools/clang/test/CodeGenDXIL/hlsl/intrinsics/buffer-agg-load-stores.hlsl

```diff
@@ -0,0 +1,231 @@
+// RUN: %dxc -DARR -DTYPE=float    -DNUM=4 -DIX=SIx -T vs_6_6 %s | FileCheck %s
```

Comment:
These test are identical to those in the preferred solution #7114. See that for more comments on them.